mcs/class/monodoc/Monodoc.Ecma/EcmaDesc.cs

```diff
@@ -263,9 +255,18 @@ void ConstructCRef (StringBuilder sb)
 			if (DescKind == Kind.Type)
 				return;
 
+			if (ExplicitImplMember != null) {
+				sb.Append ('$');
```

Comment:
without changes to the parser/grammar, not currently ... I tested, and it fails to parse crefs that contain `#`. Much of the codebase assumes that we'll be using `$`, so it would probably be a bigger job.